deployment/examples/simulator-deployment/kustomization.yaml

```
@@ -0,0 +1,14 @@
+
+metadata:
+  name: maas-production-deployment
+
+# Full production deployment
+resources:
+- ../../core-infrastructure
+- ../kustomize-templates/models/simulator
+- ../kustomize-templates/auth/api-keys
+- ../kustomize-templates/auth/token-rate-limiting
+- ../kustomize-templates/observability
+- routes.yaml
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)